package.json

```diff
@@ -14,10 +14,12 @@
     "@fortawesome/free-brands-svg-icons": "^6.0.0",
     "@fortawesome/free-solid-svg-icons": "^6.0.0",
     "@fortawesome/react-native-fontawesome": "^0.2.7",
+    "@notifee/react-native": "^3.0.1",
```

Comment:
are you using this library ?

Reply:
I discussed this yesterday with evgenyshenets9. will be deleted